db/seeds.rb

```diff
@@ -11,3 +11,15 @@
 	encrypted_password: "password1",
 	sign_in_count: 1
 )
+
+grilled_cheese = Sandwich.create(
```

Comment:
you can consider using `create!` in your `seeds.rb` file so that it errors out if validation fails